attachment_base_synchronize/models/attachment.py

```diff
@@ -56,8 +57,7 @@ def _compute_hash(self):
     @api.model
     def run_attachment_metadata_scheduler(self, domain=None):
         if domain is None:
-            domain = []
-        domain.append(('state', '=', 'pending'))
+            domain = [('state', '=', 'pending')]
```

Comment:
That is wanted indeed.
The normal way is to just run the attachment at pending state, so the cron does not need any argument. But one can make a cron only with attachment at 'failed' state for instance.
So the idea is that we can ignore the generic domain `('state', '=', 'pending')` for more flexibility.